tests/test_classes.cpp

```diff
@@ -435,4 +435,13 @@ NB_MODULE(test_classes_ext, m) {
 
     using NonCopyableVec = std::vector<NonCopyable>;
     nb::class_<NonCopyableVec>(m, "NonCopyableVec");
+
+    // test32_deference
+    struct Deferential {nb::object target; };
```

Comment:
space after `{`.

Can you rename the class from `Dereferential` (which I think is more common to express "honor" to another person) to something like `Indirect` or `HashWrapper`? Same for the test case name.

Reply:
Sure! I was thinking it was more "defer to someone else." I'll change to `HashWrapper`